controllers/openshift_controller.go

```diff
@@ -125,9 +116,8 @@ func (r *KataConfigOpenShiftReconciler) Reconcile(ctx context.Context, req ctrl.
 		return ctrl.Result{}, err
```

Comment:
nit, maybe worth mentioning in the comment above (under `if  k8serrors.IsNotFound(err) `) aso the new case it may not be found (feguregate CM modified before KataConfig creation)

Reply:
Changed the flow a bit. The status CM is created by operator to start with.